src/include/server/mir/scene/idle_hub.h

```diff
@@ -68,6 +68,12 @@ class IdleHub
 
     virtual void unregister_interest(IdleStateObserver const& observer) = 0;
 
+    /// Inhibits idling on all surfaces until resumed
```

Comment:
Idling is not really a property of “surfaces”, right?

This *probably* doesn't need to have a comment. :)